### ISSUE TITLE
motion sensor state change to "clear" ('off') do not trigger automations

### ISSUE THREAD

**Issue by mrsnyds** [kevinvincent/ha-wyzesense#122]
This issue began with the update to HA 0.108.0, and persists after updating to 0.108.1.  It is specific to the Wyzesense motion sensors.  The contact sensors do not exhibit this problem.

The issue is that when a binary_sensor for a Wyzesense motion sensor state changes to 'off', the automation that is to supposed to trigger does not fire. The state change to 'off' is correctly reflected in the front end, but will not trigger the automation as it did before.

As an example, the following triggers are for a Wyzesense motion sensor.  Both used to work before updating to 0.108.0, but now only the to: 'on' (first trigger) works. The to: 'off' (second trigger) fails to fire the automation.

```
trigger:
- platform: state
  entity_id: binary_sensor.foyer_motion
  to: 'on'
- platform: state
  entity_id: binary_sensor.foyer_motion
  to: 'off'
```

I have similar YAML in use with a Wyzesense contact sensor, and the contact sensor state changes continue to trigger automation for state changes in both directions (to 'on' or to 'off') as they did before.

Running 0.108.1 HASSIO (HA Core) on an rPi 4
Using latest wyze bridge firmware, version 0.33

PASTE BIN HERE: https://pastebin.com/4EknBcBM

**Comment by khaleddost** [comment]
I am also having this issue on 0.108.1. Motion sensors and all automations involving them are no longer working.

**Comment by davebeckster** [comment]
Similar problems.  Has happened rarely before where it has been necessary to power off the Wyze bridge and restart HA.  Restart alone usually inadequate.   It is now happening several times daily with 0.108.0.

**Comment by VarenDerpsAround** [comment]
> https://github.com/kevinvincent/ha-wyzesense/issues/121#issuecomment-610958312

**Comment by mrsnyds** [comment]
Folks, reading these comments, I'm not sure we are talking about the same thing.   My sensors all work just fine, and Home Assistant recognizes the state changes.   No dropouts or lack of response since I updated the bridge firmway.  All of that is working consistently.  

This issue I am documenting is specifically when the state of a binary_sensor based on a Wyze **_motion_** sensor changes from "on" to "off" that an automation trigger won't fire.  When the same motion sensor changes from "off" to "on", the automation will trigger as expected, but not the other way around.  Also, automations that have triggers based on binary_sensors based on Wyze **contact** (door) sensors work as expected with state changes in both directions, as before.

**Comment by davebeckster** [comment]
Sounds like you are right.  If I have your problem I have not discovered it
yet.  Out of this one.

**Comment by khaleddost** [comment]
Hi guys, I fixed the problem I was having. I updated to 0.108.1+ and then removed all the sensors and then readded them. Now everything is working again.

**Comment by mrsnyds** [comment]
I am happy to report that my issue was NOT with the Wyzesense component.  I was misreading what I saw in the UI, and was mistaken when I stated that the trigger to 'off' was not firing.  The trigger was in fact firing, but the action for to 'off' was no longer working as it did before, due to some undocumented breaking change to the Lutron Caseta integration.  

This action no longer resulted in the brightness being set to 181 ... but results in 178!

```
   service: light.turn_on
    data:
      entity_id: light.front_foyer_foyer_star
      brightness: 181
```
This was causing my condition for the turn-off script to check that the brightness was, in fact, still 181, to fail ... because the brightness is actually 178.   The Lutron Caseta component was updated, and this is a likely side-effect.
